src/Microsoft.AspNet.Mvc.Core/Routing/AttributeRoutePrecedence.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNet.Mvc.Routing
     /// </summary>
     public static class AttributeRoutePrecedence
     {
-        public static decimal Compute(RouteTemplate template)
+        public static decimal ComputeMatched(RouteTemplate template)
```

Comment:
Not asking for a doc comment with all the pomp and circumstance - just a normal comment will do.